### PR TITLE
Sourcemaps

### PR DESCRIPTION
**Pull request by Gregoor**
Issue #483

### CODE REVIEW

**Comment by Gregoor** [comment]
@mikehenrty this should be done now. Needed to add a few more gulp tasks, but didn't really see a way around it.

**Comment by nevik** [comment]
@Gregoor other than switching to webpack ;) (#508)

**Comment by Gregoor** [comment]
@nevik I'm glad someone else said it :grin:

File: gulpfile.js [hunk 1]
```diff
@@ -244,6 +262,10 @@ gulp.task('default', 'Running just `gulp`.', ['build'], () => {
   watchAndListen();
 });
 
+gulp.task('develop', 'Run everything in dev (with sourcemaps).', ['build-dev'], () => {
+  watchAndListen(true);
+});
+
```
Comment (mikehenrty) [inline gulpfile.js]: let's just make this the default for when we run `gulp`

**Comment by Gregoor** [comment]
@mikehenrty I just tested it on both and for Chrome it worked without a hick. In FF it worked after a few reloads and now I can't get it to not work.

**Comment by mikehenrty** [comment]
Great, let's do this!

**Pull request merged**